JS/inject.js

```diff
@@ -1,147 +1,150 @@
-                         <br>
-                         <br>
-                         <button style="margin-left:18px; margin-bottom: 5px;" class="tablink button byURL">By URL</button>
-                         <button class="tablink button searchBarBTN ">Search</button>
-                         <br>
-                         <br>
-                         <div id="Paste" class="container input-type">
-                             <input placeholder="Paste Youtube URL" value="https://www.youtube.com/watch?v=LkjvwtQfRw8"
-                                 class="textInput" type="text" id="vid-url" size="30">
-                             <input type="button" class="form-control action selectBTN selectButton" value="Select"><br>
-                         </div>
-                         <div id="Search" class="container input-type">
-                             <input placeholder="Search Youtube" class="textInput" type="text" id="vid-name" size="30">
-                             <input type="button" class="selectButton action searchBTN" value="Search">
-                             <br>
-                         </div>
-                         <div id="response"></div>
-                         <div class=" container light-grey padding">
-                             <button class="right button closeBTN" style="padding-top: 4px;"> Close</button>
-                         </div>
-                     </div>
-                 </div>
-             
-                 <div id=""  style="z-index:101; display:var(--display-status); position:absolute; left:0px; top:320px;" class="controls">
-                     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-                     <div class="progress-bar">
-                         <input type="range" class="slider" id="progress-bar" value="0"><br>
-                     </div>
-                     <div class="toggle-controls" style="margin-top: 10px;">
-                         <i id="play-pause" class="material-icons">pause</i>
-                         <i id="mute-toggle" class="material-icons">volume_up</i>
-                     </div>
-                     <div class="time-display" style="position: relative; top: -6px;">
-                         <p><span id="current-time">0:00</span> / <span id="duration">0:00</span></p>
-                     </div>
-             
-                     <div class=" right toggle-controls" style="margin-top: 10px;">
-                         <i id="close-window" class="material-icons">close</i>
-                         <i id="zoom-in" class="material-icons">zoom_in</i>
-                         <i id="zoom-out" class="material-icons">zoom_out</i>
-                     </div>
-                 </div>
-               `
-        document.querySelector("body").prepend(popupGroup);
-        // $(popupGroup).append(content);
-        $("body").append(controls);
-
+        var popupGroup = `
+        <div id="playbackGroup">
```

Comment:
Again, just for discussion...

Should this be broken out into a separate HTML file? Would you HTML edit handle an HTML fragment file?